docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -84,6 +96,15 @@ Security case description is sent to {ibm-r}, the existing {ibm-r} `Description`
 field is overwritten.
 ** *Comments*: Mapped to the {ibm-r} `Comments` field. When a comment is updated
 in a Security case, a new comment is added to the {ibm-r} incident.
+* For {swimlane} records:
+** *Title*: Mapped to the {swimlane} `caseName` field. When an update to a
+Security case title is sent to {swimlane}, the existing {swimlane} `caseName` field is
```

Comment:
It is not exactly the existing `caseName` field but what is mapped to the `caseName` field. It is `title` -> `caseName` -> `<what ever field the user configured>`.